docs/admin/pool_operations.md

```diff
@@ -916,6 +916,26 @@ and possibly repair a pmemobj file. As discussed in the previous section, the
 rebuild status can be consulted via the pool query and will be expanded
 with more information.
 
+## Pool-Wise Redundancy Factor (PW_RF)
+
+When there are cascading multiple failures, if the number of failed fault
+domains exceeds the PW_RF, there could be unrecoverable errors and applications
+could suffer from data loss. Especially when there is power or network outage,
+it could cause massive node/engine failures. In most cases those failures can
+be recovered and DAOS engines can be restarted and work again.
+
+Administrator can set PW_RF by environment variable "DAOS_PW_RF". When SWIM
```

Comment:
This needs to be imo a server yaml setting and not an env variable. something like:
```
port: 10001
provider: ofi+tcp
socket_dir: /tmp/daos_server
min_pool_rf: 2
```

Reply:
in 2.6.2 will add pool property for that, so need not yml file setting later.